src/clients/docs_generate.zig

```diff
@@ -671,6 +689,18 @@ const Generator = struct {
                     language.developer_setup_pwsh_commands,
                 }),
             );
+
+            mw.header(4, "`InvalidFormat` error");
```

Comment:
😢

Reply:
Fixed by #960, we can remove this instruction.

Reply:
Done! Thanks!